packages/babel-preset/non-standard-plugins.js

```diff
@@ -16,14 +16,19 @@ module.exports = function shopifyNonStandardPlugins(options = {}) {
     // proposal-class-properties must be used in loose mode
     // see https://babeljs.io/docs/en/babel-plugin-proposal-decorators#note-compatibility-with-babel-plugin-proposal-class-properties
     plugins.push(
-      require.resolve('@babel/plugin-syntax-nullish-coalescing-operator'),
```

Comment:
syntax support for nullish coalescing / optional chaining are already added by babel-preset-env, so no need to reference them here or in our package.json